conf/layer.conf

```diff
@@ -7,7 +7,7 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 
 BBFILE_COLLECTIONS += "meta-qcom-hwe"
 BBFILE_PATTERN_meta-qcom-hwe = "^${LAYERDIR}/"
-BBFILE_PRIORITY_meta-qcom-hwe = "10"
+BBFILE_PRIORITY_meta-qcom-hwe = "6"
```

Comment:
IMO, it is best if priority is set to 5 instead of 6, same as base layers. 
As this layer can potentially override some of the recipes from base layers, having a higher BBFILE_PRIORITY would override DEFAULT_PREFERENCE of recipes, which is totally unwanted.